style.css

```diff
@@ -19,3 +19,21 @@ body {
   border-top: 1px solid #007bff;
   border-bottom: 3px solid orangered;
 }
+
+.switch-button {
+  appearance: none;
+  border: none;
+  background: #007bff;
```

Comment:
this matches the document background colour and therefore does not appear like a button.